src/backend/dx12/src/command.rs

```diff
@@ -254,6 +254,7 @@ struct Copy {
 
 #[derive(Clone)]
 pub struct CommandBuffer {
+    device: ComPtr<d3d12::ID3D12Device>,
```

Comment:
Note: this collides with #2001, which has a little more comprehensive approach to device pointer sharing
cc @msiglreith